etc/kayobe/ansible/templates/os_capacity-clouds.yml.j2

```diff
@@ -1,10 +1,13 @@
 clouds:
   openstack:
     auth:
-      auth_url: "{{ secrets_os_exporter_auth_url }}"
-      application_credential_id: "{{ secrets_os_exporter_credential_id }}"
-      application_credential_secret: "{{ secrets_os_exporter_credential_secret }}"
+      auth_url: "{{ secrets_os_capacity_auth_url }}"
```

Comment:
Is this a secret?

Reply:
Doesn't really need to be since it's just the keystone auth url, this config is deprecated in #829 though.

Reply:
So can you change it to use a Kolla Ansible variable?